bcda/encryption/encryption_test.go

```diff
@@ -38,14 +37,14 @@ func (s *EncryptionTestSuite) TestEncryptBytes() {
 	// Make a random String for encrypting
 	testBytes := []byte(uuid.NewRandom().String())
 	// Encrypt the sting and get the key back
-	encryptedBytes, encryptedKey, err := EncryptBytes(auth.GetATOPublicKey(), testBytes, "TEST")
```

Comment:
You can remove lines 32/3 above.  The old structure was finding these keys from 2 different levels of code and now that value is what is in the YML file and should not need to be overridden here.

os.Setenv("ATO_PUBLIC_KEY_FILE", "../../shared_files/ATO_public.pem")
os.Setenv("ATO_PRIVATE_KEY_FILE", "../../shared_files/ATO_private.pem")

Reply:
This PR is about moving the models, not fixing configuration related to a different PR and code change. For traceability, I think it's important to keep PRs focused on the work of the ticket, and the changes you describe here should be in a separate ticket.